journal-of-sport-and-health-science.csl

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
```

Comment:
since the journal is English only, you'll want to add `default-locale="en-US"` here